src/app/game-state/main-loop.service.ts

```diff
@@ -102,26 +110,39 @@ export class MainLoopService {
       const newTime = new Date().getTime();
       const timeDiff = newTime - this.lastTime;
       this.lastTime = newTime;
-      //this should be around 1, but may vary based on browser throttling
+      // this should be around 1, but may vary based on browser throttling
       let ticksPassed = timeDiff / TICK_INTERVAL_MS; 
+      const setTPS = this.getTPS(this.tickDivider) / 1000 * TICK_INTERVAL_MS;
+      const topTPS = this.getTPS(this.topDivider) / 1000 * TICK_INTERVAL_MS;
```

Comment:
Base banked ticks on fastest speed available, because if players weren't offline or paused they'd be going as fast as possible most times.